packages/menu/src/Menu.js

```diff
@@ -50,12 +54,14 @@ export default class Menu extends Component {
   };
 
   static defaultProps = {
-    multiple: false
+    multiple: false,
+    unselect: false
```

Comment:
per our conversation, i believe `unselect` should be `true` by default as currently all users of this component can unselect options, and if you were to make that `false` now you risk potentially breaking it for everyone who upgrades.